2020/hamming-codes/spanish/sentence_translations.json

```
@@ -623,7 +623,7 @@
  {
   "input": "This is only 1 out of 4 parity checks that we'll do.",
   "model": "nmt",
-  "translatedText": "Esta es sólo 1 de cada 4 comprobaciones de paridad que haremos.",
+  "translatedText": "Esto es sólo 1 de los 4 controles de paridad que haremos.",
```

I think "este" (or "éste" if using older spelling rules) sounds slightly better, but it could be a regional preference of mine (if so, please disregard):

```suggestion
  "translatedText": "Este es sólo 1 de los 4 controles de paridad que haremos.",
```